rpki-validator/src/main/java/net/ripe/rpki/validator3/background/ValidationScheduler.java

```diff
@@ -95,28 +123,12 @@ public boolean scheduledTrustAnchor(TrustAnchor trustAnchor) {
             return false;
         }
         try {
-            return scheduler.checkExists(TrustAnchorValidationJob.getJobKey(trustAnchor)) &&
-                    scheduler.checkExists(CertificateTreeValidationJob.getJobKey(trustAnchor));
+            return scheduler.checkExists(TrustAnchorValidationJob.getJobKey(trustAnchor));
         } catch (SchedulerException e) {
             return false;
         }
     }
 
-    public void removeTrustAnchor(TrustAnchor trustAnchor) {
```

Comment:
Unused/dead code?

Reply:
Yeah, it wasn't actually used, it's some boyscouting.